gradle/publish.gradle

```diff
@@ -1,6 +1,8 @@
 apply plugin: 'signing'
 apply plugin: 'io.codearte.nexus-staging'
 
+def augmentedSbom = artifacts.add('sbom', file('artifacts/ssdlc/mongo-jdbc-driver.augmented.sbom.json'))
```

Comment:
Take a look at this page : https://docs.gradle.org/current/userguide/working_with_files.html
I think you will need to use [Project.getRootDir()](https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:rootDir).
My understanding is that otherwise it is the path relative to where the gradle script is

Reply:
For reference, `$rootDir`, is the same as `Project.getRootDir()`.